packages/sdk-browsersigner/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@dfns/sdk-browsersigner",
```

Comment:
nit: let's just go with `sdk-browser`, we can include more helpers besides signers in the future maybe.